### ISSUE TITLE
SNP and copy number conversion?

### ISSUE THREAD

**Issue by Neato-Nick** [PoisonAlien/varscan_accessories#3]
Hi,

Does these scripts also work when converting copy number findings from VarScan? I see from the other, well-used thread that they work for indels and SNPs at least.

Thanks!

**Comment by PoisonAlien** [comment]
Hi,

For now it works only for VCF or native (SNP and INDELS) files generated by varscan somatic (and processSomatic) command.

**Comment by Neato-Nick** [comment]
Great, thanks for letting me know!

**Comment by Neato-Nick** [comment]
Do you have any suggestions for processing of somatic CNVs called through VarScan then?

**Comment by PoisonAlien** [comment]
Can you explain a little bit ? Process how ? You can perform segmentation with DNACopy. How many samples do you have ? I find gatk4 cnv protocol to work the best!

**Comment by Neato-Nick** [comment]
I'm pretty new to the variant-finder game, so I apologize for being less-than-understandable sometimes. We are aiming to call CNVs with multiple programs and then rate the calls by consensus, so we are also using the GATK CNV protocol! So to use VarScan as another program, I could call the CNVs with it, then put the output into R and segment with DNACopy?

Edit: Just found this BioStars post: https://www.biostars.org/p/10860/. Looks like that's right. However, it doesn't look like DNACopy outputs to VCF. If I want VCF descriptions, is the calling straight out of VarScan's copynumber what I want?

**Comment by PoisonAlien** [comment]
Hi ,
No problem and you're correct. Use .copynumber file generated by `varScan copynumber` command to perform segmentation, and DNCopy doesn't output vcf file neither does varscan copynumber. VCF is only for storing variants not for copy number segments.

If you want feel free to use this [script](https://gist.github.com/PoisonAlien/11e49f43b7d899516d13b31eab767435) which takes varscan copynumber output, performs segmentation, writes CBS file and generates a plot. You can open CBS file in IGV too.

Run it from command line:

```r
$Rscript dnaCopyScan.r foo.copynumber
```

**Comment by Neato-Nick** [comment]
Wonderful. I'll use that script for sure. Is there a way I can get the original .copynumber file into VCF? It looks like segmentation has its own file format that needs preserving, but it would be nice to get the copy number discoveries into the same VCF file at the end as my indels and SNPs.

**Comment by PoisonAlien** [comment]
yes segmentation files are usually stored as CBS file which is generated by DNACopy. Unfortunately it is not possible to store copynumber output as vcf.
